src/auth/token.rs

```diff
@@ -40,6 +40,7 @@ pub struct TokenCreateRequest {
     pub name: String,
     pub description: String,
     pub expires_at: Option<DateTime<Utc>>,
+    pub namespace_key: String,
```

Comment:
```suggestion
    pub namespace_key: Option<String>,
```

Should be optional I think